modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/FormRenderer/FormRenderer.soy

```diff
@@ -16,70 +17,72 @@
 	{@param? strings: map<string, string>}
 	{@param? submitLabel: string}
 
-	{let $currentPaginationMode: $paginationMode ?: 'wizard' /}
-	{let $currentPage: $activePage ?: 0 /}
-	{let $total: length($pages) /}
+	{if $displayable}
+		{let $currentPaginationMode: $paginationMode ?: 'wizard' /}
+		{let $currentPage: $activePage ?: 0 /}
+		{let $total: length($pages) /}
 
-	{let $classes kind="text"}
-		lfr-ddm-form-container position-relative
+		{let $classes kind="text"}
+			lfr-ddm-form-container position-relative
 
-		{if not $editable}
-			{sp}ddm-user-view-content
-		{/if}
-	{/let}
+			{if not $editable}
+				{sp}ddm-user-view-content
+			{/if}
+		{/let}
 
-	<div class="{$classes}">
-		{if $currentPaginationMode == 'tabbed'}
-			{let $page: $pages[$currentPage] /}
+		<div class="{$classes}">
+			{if $currentPaginationMode == 'tabbed'}
+				{let $page: $pages[$currentPage] /}
 
-			{call PageRenderer.render}
-				{param activePage: $activePage /}
-				{param editable: $editable /}
-				{param events: [
-					'fieldBlurred': $_handleFieldBlurred,
-					'fieldClicked': $_handleFieldClicked,
-					'fieldEdited': $_handleFieldEdited,
-					'fieldFocused': $_handleFieldFocused
-				] /}
-				{param key: 'pageRenderer' + $activePage /}
-				{param page: $page /}
-				{param pageIndex: $activePage /}
-				{param pages: $pages /}
-				{param paginationMode: $currentPaginationMode /}
-				{param readOnly: $readOnly /}
-				{param ref: 'pageRenderer' + $activePage /}
-				{param showSubmitButton: $showSubmitButton != false /}
-				{param spritemap: $spritemap /}
-				{param strings: $strings /}
-				{param submitLabel: $submitLabel ?: 'submit' /}
-				{param total: $total /}
-			{/call}
-		{else}
-			{foreach $page in $pages}
 				{call PageRenderer.render}
 					{param activePage: $activePage /}
 					{param editable: $editable /}
-					{param editingLanguageId: $editingLanguageId /}
```

Comment:
@linolaoj, why did you need to remove the editing language just for this case?

Reply:
@natocesarrego , this was actually an accident. Sorry I missed that. I'll add it back.